src/chain_specs/aldebaran.json

```diff
@@ -386,19 +386,19 @@
         "creator": "5GGesvoesHcWwuvTJhLkfoxd39DwaS2K49PENQL8f6Ax91D2",
         "signers": [
           {
-            "AccountKey": "0x9870910d65ccb11af1c85d8ed1a17bc2229bbc08edf44c0dc7865e0edbef8b35"
+            "Account": "0x9870910d65ccb11af1c85d8ed1a17bc2229bbc08edf44c0dc7865e0edbef8b35"
```

Comment:
Past chain specs should not be changed. This file is present only for "preservation".

Reply:
OK, I'll roll it back.